deps.edn

```diff
@@ -193,23 +192,26 @@
   ;;    (dev/start!)
   :dev
   {:extra-deps
-   {clj-http-fake/clj-http-fake  {:mvn/version "1.0.3"
+   {
+    clj-http-fake/clj-http-fake  {:mvn/version "1.0.3"
                                   :exclusions  [slingshot/slingshot]}
-    clj-kondo/clj-kondo          {:mvn/version "2022.08.03"}                    ; this is not for RUNNING kondo, but so we can hack on custom hooks code from the REPL.
+    clj-kondo/clj-kondo          {:mvn/version "2022.08.03"}                ; this is not for RUNNING kondo, but so we can hack on custom hooks code from the REPL.
     cloverage/cloverage          {:mvn/version "1.2.2"}
-    djblue/portal                {:mvn/version "0.30.0"}
+    com.gfredericks/test.chuck   {:mvn/version "0.2.13"}                    ; generating strings from regexes (useful with malli)
+    djblue/portal                {:mvn/version "0.30.0"}                    ; ui for inspectinv values
     eftest/eftest                {:mvn/version "0.5.9"}
     io.github.camsaul/humane-are {:mvn/version "1.0.2"}
-    jonase/eastwood              {:mvn/version "1.2.2"}
-    methodical/methodical        {:mvn/version "0.15.0.1"}
+    jonase/eastwood              {:mvn/version "1.2.2"}                     ; inspects namespaces and reports possible problems using tools.analyzer
+    lambdaisland/deep-diff2      {:mvn/version "2.3.127"}                   ; way better diffs
```

Comment:
TIL; where/how is this used?

Reply:
aha https://github.com/metabase/metabase/blob/d18e3da72b346124ddfd320457ac7d1b1ab5ceb7/dev/src/dev/debug_qp.clj#L173-L194